src/styles/index.scss

```diff
@@ -1,3 +1,139 @@
+$day-size: 100px;
+$border: 1px;
+$grey: #eee;
+$font-size: 30px;
+$gap: 1px;
+$padding: 10px;
+
+// DAYS
+
+$days: (
+  mon: 'Monday',
+  tue: 'Tuesday',
+  wed: 'Wednesday',
+  thu: 'Thursday',
+  fri: 'Friday',
+  sat: 'Saturday',
+  sun: 'Sunday',
+);
+$months: (28, 29, 30, 31);
+
 body {
   margin: 0;
 }
+
+.container {
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, sans-serif;
+  font-size: $font-size;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  gap: $gap;
+  width: 706px;
+  padding: $padding;
+  display: flex;
+
+  &__day {
+    background-color: $grey;
+    width: $day-size;
+    height: $day-size;
+    border: $border solid #000;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
+    display: flex;
+    box-sizing: border-box;
+  }
+
+  &__day:first-child {
+    content: '';
```

Comment:
The use of `content: '';` in the `.calendar__day:first-child` selector is incorrect. The `content` property is used with pseudo-elements like `::before` or `::after`, not directly on elements.